src/pkgdefaults.jl

```diff
@@ -231,6 +231,22 @@ substrate per s.
 \nPermyriad, a unit meaning parts per ten thousand. Printed as \"‱\".
 \nDimension: [`Unitful.NoDims`](@ref)."
 @unit pertenthousand  "‱"    Pertenthousand  1//10000           false
+"    Unitful.ppm
+\nPermillion, a unit meaning parts per million.
+\nDimension: [`Unitful.NoDims`](@ref)."
+@unit ppm             "ppm"  Permillion      1//1000000         false
+"    Unitful.ppb
+\nPerbillion, a unit meaning parts per billion.
```

Comment:
```suggestion
\nPerbillion, a unit meaning parts per billion (in the short-scale sense), i.e., 10^-9.
```
Since billion, trillion, and quadrillion are ambiguous (https://en.wikipedia.org/wiki/Long_and_short_scales), it would be good to mention the actual values.